packages/material-ui/src/Modal/ModalManager.ts

```diff
@@ -52,11 +58,16 @@ function findIndexOf(containerInfo, callback) {
   return idx;
 }
 
-function handleContainer(containerInfo, props) {
-  const restoreStyle = [];
-  const restorePaddings = [];
+function handleContainer(containerInfo: Container, props: ManagedModalProps) {
+  const restoreStyle: Array<{
+    /**
+     * CSS property name (HYPHEN CASE) to be modified.
```

Comment:
Would it be clearer? 🍡🍢
```suggestion
     * CSS property name (kebab-case) to be modified.
```

Reply:
hyphen case matches the MDN docs:

> propertyName is a DOMString representing the CSS property name (hyphen case) to be modified.

-- https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty

Upper case for visibility.

Reply:
Is MDN easier to understand?

Reply:
Yes.